libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/DummyExoMediaDrm.java

```diff
@@ -156,4 +156,14 @@ public CryptoConfig createCryptoConfig(byte[] sessionId) {
   public @C.CryptoType int getCryptoType() {
     return C.CRYPTO_TYPE_UNSUPPORTED;
   }
+
+  @Override
+  public void removeOfflineLicense(byte[] keySetId) {
+    // Do nothing
+  }
+
+  @Override
+  public List<byte[]> getOfflineLicenseKeySetIds() {
+    return null;
```

Comment:
This method is not annotated `@Nullable`, so it's implicitly non-null. Probably best to return an empty list here.

Reply:
Thanks for reviewing. Updated.